examples/hybrid/define_tc_quicklook_profiles.jl

```diff
@@ -4,7 +4,7 @@ using Plots
 function plot_tc_profiles(folder, hdf5_filename)
 
     input_filename = joinpath(folder, hdf5_filename)
-    output_filename = joinpath(folder, "final_profiles.png")
+    output_filename = joinpath(folder, "____________________________________final_profiles.png")
```

Comment:
🙀

Reply:
lol, I have no idea how else to make the profiles plots stick out from all the animations

Reply:
Fair point

Reply:
The formatter also does not like me 😹